tutorials/robot-marbles-part-1/robot-marbles-part-1.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p><code>simulation_parameters</code> should be preprocessed with <code>config_sim</code> and the result then passed to <code>append_configs</code> . See <a href="https://github.com/cadCAD-org/cadCAD/tree/master/documentation#simulation-properties" target="_blank">https://github.com/cadCAD-org/cadCAD/tree/master/documentation#simulation-properties</a></p><pre class="ql-syntax" spellcheck="false">from cadCAD.configuration import append_configs
from cadCAD.configuration.utils import config_sim

sim_config_dict = {
&nbsp;&nbsp;"N": ...,
&nbsp;&nbsp;"T": range(...),
&nbsp;&nbsp;"M": ...
}

c = config_sim(sim_config_dict)

append_configs(
&nbsp;&nbsp;...
&nbsp;&nbsp;sim_configs = c # Simulation Properties
)
</pre><p> </p><p>In some situations it works without the need for the preprocessing, and this might be one of those, but IMO the tutorials should follow the best practices so that when readers modify them slightly things don't break :-) This comment applies to all the notebooks and cells where <code>simulations_parameters</code>  is defined.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cadCAD-org/cadCAD/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>